package.json

```diff
@@ -8,10 +8,12 @@
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
     "firebase": "^7.14.3",
+    "moment": "^2.26.0",
     "node-sass": "^4.14.1",
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-cookie": "^4.0.3",
+    "react-datepicker": "^2.16.0",
```

Comment:
this is fine we need to manage native datepickers so probably we can use a date input.
But this is perfect right now

Reply:
Done in Last Commit